hasher-matcher-actioner/hmalib/common/content_sources.py

```diff
@@ -6,15 +6,15 @@
 import typing as t
 
 
-class ImageSource:
+class ContentSource:
```

Comment:
Naming nit: We use "Source" as a [defined term](https://github.com/facebook/ThreatExchange/wiki/Glossary#fetcher) elsewhere in HMA. Can we use another word here? "Origin" maybe?

Reply:
Sure. How about `ContentStorage`? Subclasses would be `S3ContentStorage` and `URLContentStorage`.